src/app/settings/page.tsx

```
@@ -0,0 +1,893 @@
+        data.chatModelProviders[chatModelProvider]?.length > 0
+          ? data.chatModelProviders[chatModelProvider][0].name
+          : undefined) ||
+        '';
+      const embeddingModelProvider =
+        localStorage.getItem('embeddingModelProvider') ||
+        defaultEmbeddingModelProvider ||
+        '';
+      const embeddingModel =
+        localStorage.getItem('embeddingModel') ||
+        (data.embeddingModelProviders &&
+          data.embeddingModelProviders[embeddingModelProvider]?.[0].name) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)